docs/getting-started/using-sqlfmt.mdx

```diff
@@ -42,4 +49,40 @@ sqlfmt --diff .
 sqlfmt can also format code passed through standard input (`stdin`) by passing `-` as the files argument. The formatted code will be output to `stdout` (all other output from sqlfmt is routed to `stderr`):
 ```bash
 echo "select 1" | sqlfmt -
-```
\ No newline at end of file
+```
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+sqlfmt is also available as a Dockerized container. This allows for formatting without having Python installed. It currently does not support the `check` and `diff` functionality.
+
+To format the files in the current directory and it's subdirectory:
+
+```bash
+docker run 
+    -v $(pwd):/src 
+    ghcr.io/tconbeer/sqlfmt:latest
+```
+
+In this example `$(pwd)` will return the current working directory. All files in this directory will be mounted to the `/format` folder from where they will be formatted.
+
+To format specific files multiple mounts can be passed to the container.
+
+```bash
+docker run 
+    -v {path to file one}:/src/file_one 
+    -v {path to file two}:/src/file_two
+    ghcr.io/tconbeer/sqlfmt:latest
+```
+
+To pass along the configuration parameters described in [configuring sqlfmt](./configuring-sqlfmt.md) an environment variable can be passed along:
```

Comment:
If the mounted directory includes a `pyproject.toml` file, sqlfmt will use the configuration set in that file.